include/ignition/math/Vector3.hh

```diff
@@ -97,9 +97,10 @@ namespace ignition
       /// \return the distance
       public: T Distance(const Vector3<T> &_pt) const
       {
-        return sqrt((this->data[0]-_pt[0])*(this->data[0]-_pt[0]) +
+        return static_cast<T>(sqrt(
+                    (this->data[0]-_pt[0])*(this->data[0]-_pt[0]) +
                     (this->data[1]-_pt[1])*(this->data[1]-_pt[1]) +
-                    (this->data[2]-_pt[2])*(this->data[2]-_pt[2]));
+                    (this->data[2]-_pt[2])*(this->data[2]-_pt[2])));
```

Comment:
windows is still giving warnings about this function

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_math-ci-pr_any-windows7-amd64&build=981)](https://build.osrfoundation.org/job/ignition_math-ci-pr_any-windows7-amd64/981/) https://build.osrfoundation.org/job/ignition_math-ci-pr_any-windows7-amd64/981/msbuild/new/

if we didn't mind the memory allocation, I think we could use `return (*this - _pt).Length();`

Reply:
Ahh that was a typo in the test: 9562793765d82feefd0d17446b3d7199d8c74aa0